src/basic_recipes/basic_recipes.jl

```diff
@@ -580,27 +580,32 @@ end
 
 """
     band(x, ylower, yupper; kwargs...)
+    band(lower, upper; kwargs...)
 
 Plots a band from `ylower` to `yupper` along `x`.
 
 ## Theme
 $(ATTRIBUTES)
 """
-@recipe(Band, x, ylower, yupper) do scene
+@recipe(Band, lowerpoints, upperpoints) do scene
     Theme(;
         default_theme(scene, Mesh)...,
         color = RGBAf0(1.0,0,0,0.2)
     )
 end
 
+convert_arguments(::Type{<: Band}, x, ylower, yupper) = [Point2f0.(x, ylower); Point2f0.(x, yupper)]
```

Comment:
```suggestion
convert_arguments(::Type{<: Band}, x, ylower, yupper) = (Point2f0.(x, ylower), Point2f0.(x, yupper))
```
Not sure, but this might be correct looking at line 606